store/contracts.ts

```diff
@@ -1,15 +1,217 @@
-import { Module, VuexModule } from "vuex-module-decorators"
+import { Action, Module, Mutation, VuexModule } from "vuex-module-decorators"
+import { ethers } from "ethers"
+import LSPAbi from "~/abis/LSP.json"
+import { getCurrentProvider } from "~/store/web3"
 import { LSPConfiguration } from "~/types"
 
+const abis: Record<string, Array<string>> = require("~/abis")
+const addresses: Record<string, string> = require("~/addresses.json")
+
+type SyntheticTokenContractMapping = {
```

Comment:
let's put these types inside of our type directory and maybe create an alias for something like the Record types being used in this file.

Reply:
Done